openshift/templates/backup/backup-build.yaml

```diff
@@ -0,0 +1,72 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  name: "${NAME}-build-template"
+  creationTimestamp:
+objects:
+  - kind: ImageStream
+    apiVersion: v1
+    metadata:
+      name: "${NAME}"
+  - kind: BuildConfig
+    apiVersion: v1
+    metadata:
+      name: "${NAME}"
+      labels:
+        app: "${NAME}"
+    spec:
+      triggers:
+        - type: ImageChange
+        - type: ConfigChange
+      runPolicy: Serial
+      source:
+        type: Git
+        git:
+          uri: "${GIT_REPO_URL}"
+          ref: "${GIT_REF}"
+        contextDir: "${SOURCE_CONTEXT_DIR}"
+      strategy:
+        type: Docker
+        dockerStrategy:
+          dockerfilePath: "${DOCKER_FILE_PATH}"
+      output:
+        to:
+          kind: ImageStreamTag
+          name: "${NAME}:${OUTPUT_IMAGE_TAG}"
+parameters:
+  - name: NAME
+    displayName: Name
+    description:
+      The name assigned to all of the resources.  Use 'backup-{database name}'
+      depending on your database provider
+    required: true
+    value: backup-mssql
+  - name: GIT_REPO_URL
+    displayName: Git Repo URL
+    description: The URL to your GIT repo.
+    required: true
+    value: https://github.com/devinleighsmith/backup-container.git
```

Comment:
Might want to change the default value here

Reply:
Agree